Game/model/model.gd

```diff
@@ -305,27 +305,39 @@ func get_storage_cap(player_id: int, type: Types.Item) -> float:
 	else:
 		return 0.0
 
+
 ## Set the heat data for the given player to the data given.
 func add_heat_data_at(
-	player_id: int, grid_position: Vector2i, heat: float, heat_capacity: float
+	player_id: int,
+	grid_position: Vector2i,
+	heat: float,
+	heat_capacity: float,
+	heat_state: Types.HeatState
 ) -> void:
-	add_heat_data_for_all_players.rpc(player_id, grid_position, heat, heat_capacity)
+	add_heat_data_for_all_players.rpc(player_id, grid_position, heat, heat_capacity, heat_state)
```

Comment:
I would probably assert that we're the server before sending this, to make it clear this should only be called by the server.